src/boot.py

```diff
@@ -0,0 +1,20 @@
+"""
+boot.py
+
+This file is executed before the USB connection is setup so we can alter what
+USB devices are created.
+"""
+import usb_cdc
+import storage
+
+# Make USB mount read-only (to host), this can only be done in boot.py
+storage.remount('/', readonly=False)
```

Comment:
> Make USB mount read-only

`readonly=False`

Eh?

Reply:
The virtual drive can only be writeable from one location (MCU or host), the read-only argument is from the perspective of the MCU.

Yes, the comment needs improvement.

Reply:
bef71f6b627e579cfcf261b9e205bf0b7bf3cf97